src/program/snabbvmx/lwaftr/lwaftr.lua

```diff
@@ -0,0 +1,169 @@
+module(..., package.seeall)
+
+local S = require("syscall")
+local config = require("core.config")
+local lib = require("core.lib")
+local lwtypes = require("apps.lwaftr.lwtypes")
+local setup = require("program.snabbvmx.lwaftr.setup")
+local shm = require("core.shm")
+
+local function show_usage (exit_code)
+   print(require("program.snabbvmx.lwaftr.README_inc"))
+   main.exit(exit_code)
+end
+
+local function fatal (msg)
+   print(msg)
+   main.exit(1)
+end
+
+local function file_exists (path)
```

Comment:
This same function exists in a dozen other places, some with TODO that mention it. Isn't it time to stop duplicating it already?